backend/utils.py

```diff
@@ -32,12 +37,37 @@ def commify(n):
     return f'{n:,}'
 
 
+def call_github_action(event_type: str) -> Response:
```

Comment:
@Sigfried Created this new function. Now used in two places: (i) the current refresh button implementation, (ii) during the db refresh, if it encounters data it can't fetch, it will trigger the solution in this PR: `resolve_fetch_failures.py`

Called like: `call_github_action('resolve-fetch-failures')`

Because of the change with 'i', I want to double check that this refactor didn't break the button. I don't have time for that today, but I've added it to my list of todo's.

### Coordinated deployments & testing
Though I also wonder if we should, as a team, do coordinated deployments. Like we can make an issue and name/tag it with the TermHub version we're deploying. Then list out all of the things that we want to test and make sure work when we're deploying to dev. Then only after all those things work do we deploy to prod.